src/pages/Home.js

```diff
@@ -43,6 +43,18 @@ const FullScreenGrid = styled(Grid)`
   overflow: hidden; //TODO fix map overflow
 `
 
+const GetProjectTypeColour = (project) => {
+  const colours = {
+    'Feasability Study': '#27847A',
+    'Funding/Competition': '#80CC72',
+    'Testing & certification': '#B6EFA0',
+    'Innovation programme': '#DFE667',
+    'R&D facility': '#C8B88A',
+    'Government strategy': '#F1DDDF',
+  }
+  return colours[project] ?? '#27847A'
+}
+
```

Comment:
nice for using '??' operator. i would not sure if i would have gone a function way. it could be just an object and default is already set i think, not sure tho